src/main/java/sirius/kernel/async/ParallelTaskExecutor.java

```diff
@@ -68,27 +85,36 @@ public boolean submitTask(Runnable task) {
         });
     }
 
+    /**
+     * Determines whether the executor is still active.
+     *
+     * @return {@code true} if the executor is still active, {@code false} otherwise
+     */
+    public boolean isActive() {
+        return Boolean.TRUE.equals(isActiveSupplier.get());
```

Comment:
Ist das so nicht schöner (oder wo ist der Vor/Nachteil?)
```suggestion
        return isActiveSupplier.get().booleanValue();
```